instance/migrations/0114_auto_20190227_1534.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```

Comment:
@giovannicimolin, can the file be named meaningfully instead of using an auto-generated name?